svelte.config.js

```diff
@@ -0,0 +1,22 @@
+/**********************************************************************
```

Comment:
remark: I am not a big fan of having it in the `packages/` folder, in't there a better place of it? maybe we should have it at the root and name it `shared.svelte.config.js`